WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -142,12 +154,18 @@ class MediaLibraryViewController: WPMediaPickerViewController {
 
             var barButtonItems = [UIBarButtonItem]()
 
-            if blog.userCanUploadMedia && assetCount > 0 {
-                let addButton = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(addTapped))
+            if blog.userCanUploadMedia {
+                addButton.spotlightOffset = Constants.addButtonSpotlightOffset
+                let config = UIImage.SymbolConfiguration(textStyle: .body, scale: .large)
+                let image = UIImage(systemName: "plus", withConfiguration: config) ?? .gridicon(.plus)
```

Comment:
We were using the system's default button. Since I had to manually create the button to add the spotlight view, I tried (to the best of my ability) to maintain the same look as before. I found that the plus grid icon was a bit thicker than the system icon.

Grid Icon | System
-|-
![GridIcon](https://user-images.githubusercontent.com/25306722/167327586-2952d67a-10b4-4e0d-adba-d173332ddfc8.png)|![System](https://user-images.githubusercontent.com/25306722/167327578-934d07fd-aeb5-4b58-a4f0-a58c5aa77aa2.png)